### PR TITLE
*: move the functions that only used in tests

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve?

Issue Number: ref #28720 

Problem Summary:

Discovered by disabling the tests linters(`--tests=false`):
```
tools/bin/golangci-lint run -v --config .golangci.yml --tests=false
```
Because some of the functions are only used by tests, the `unused` linter complains after filtering the test files.

### What is changed and how it works?

This PR removes some unused fields and rename the file.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by hawkingrei — APPROVED** [review]

**Comment by hawkingrei** [comment]
LGTM

File: dumpling/export/sql_type.go [hunk 1]
```diff
@@ -11,10 +11,9 @@ import (
 var colTypeRowReceiverMap = map[string]func() RowReceiverStringer{}
 
 var (
-	nullValue           = "NULL"
-	quotationMark       = []byte{'\''}
-	twoQuotationMarks   = []byte{'\'', '\''}
-	doubleQuotationMark = []byte{'"'}
```
Comment (bb7133) [inline dumpling/export/sql_type.go:17]: Why removing this const?

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**